src/components/DayPicker.jsx

```diff
@@ -301,15 +316,20 @@ export default class DayPicker extends React.Component {
     const {
       navPrev,
       navNext,
+      orientation,
     } = this.props;
 
+    const onNextMonthClick = orientation === VERTICAL_SCROLLABLE
+      ? this.multiplyScrollableMonths
+      : this.onNextMonthClic;
```

Comment:
`this.onNextMonthClick`, you're missing a k.

Also we prefer an `if... then` over a multiline ternary.